Fix building guide link. (#151)
maplibre/maplibre-rs: README.md

```

> More information about building for different platforms can be

Reviewer: ```suggestion
More information about running the demos on different platforms can be
```
